docs/releases/2.7.0.rst

```diff
@@ -0,0 +1,235 @@
+=====================================
+Welcome to the new Pootle 2.7.0 alpha
+=====================================
+
+*Not yet released*
+
+This is the first release of Pootle that combines the work of Evernote and
+Translate.
+
+Changes in Requirements
+=======================
+- Django >= 1.6.10 < 1.7
+- `Translate Toolkit <http://toolkit.translatehouse.org/download.html>`_ >=
+  1.12.0
+- Python >= 2.6
+- Redis - latest stable release.
+- Raven 5.0.0 or higher is now recommended for Sentry support on production
+  setups. While older versions should still work, they may encounter problems
+  in newer versions of Django or Python.
+
+
+Major Changes
+=============
+
+These are by no means exhaustive, check the `git log
+<https://github.com/translate/pootle/compare/stable%2F2.5.1...master>`_
+for more details.
+
+
+- Statistics calculations:
+
+  - Major rewrite of the way statistics are handled. RQ is used to manage the
+    calculation of invalidated statistics.  On the frontend side, stats are now
+    loaded asynchronously, thus any pending calculations no longer block page
+    loads.
+  - Custom word counting calculation method.
+
+
+- Big revamp of UI:
+
+  - We now have a wide stats table that shows the same data in the different
+    levels. This affects several features which are now relocated in a sidebar:
+    actions, instructions or current goal.
+  - New navigation scheme using dropdowns.
+  - Removed zoom feature. Now users should rely on their browsers.
+  - *All Projects* page now shares the same browsing table as available in any
+    other overview page.
+  - When there are failing checks, overview tables now display the number of
+    units which have failing checks, not the total number of failing checks.
+  - Table sorting is now remembered across overview pages, and not separately
+    in project, language and translation project pages.
+  - Major speed improvements when calculating last action information.
+  - User actionable items in the navbar have been moved to a drop-down.
+  - When selecting languages, redirect logic is smarter now.
+  - Different last activity messages for new translations and edits.
+  - Added landing page for anonymous users.
```

Comment:
You can merge this with the next point, something like:
*— New welcome page for anonymous users, which displays the top scorers over the last 30 days.*